contributing/community/reviews.rst

```diff
@@ -0,0 +1,190 @@
+step.
+
+Create a GitHub Account
+-----------------------
+
+Symfony uses GitHub_ to manage bug reports and pull requests. If you want to
+do reviews, you need to `create a GitHub account`_ and log in.
+
+The Bug Report Review Process
+-----------------------------
+
+A good way to get started with reviewing is to pick a bug report from the
+`bug reports in need of review`_.
+
+The steps for the review are:
+
+1. **Is the Report Complete?**
+
+   Good bug reports contain a link to a fork of the `Symfony Standard Edition`_
+   (the "reproduction project") that reproduces the bug. If it doesn't, the
+   report should at least contain enough information and code samples to
+   reproduce the bug.
+
+2. **Reproduce the Bug**
+
+   Download the reproduction project and test whether the bug can be reproduced
+   on your system. If the reporter did not provide a reproduction project,
+   create one by forking_ the `Symfony Standard Edition`_. Reproduce the bug
+   with the instructions given by the reporter.
+
+3. **Update the Issue Status**
+
+   At last, add a comment and update the status of the bug report. **Thank the
+   reporter for reporting the bug**. Include the line ``Status: <status>`` in
+   your comment to update the status to one of the following:
+
+   **Needs Work** If the bug does not contain enough information to be
+   reproduced, explain what information is missing and move the report to this
+   status.
+
+   **Works for me** If the bug *does* contain enough information to be
+   reproduced but works on your system, or if the reported bug is a feature and
+   not a bug, provide a short explanation and move the report to this status.
+
+   **Reviewed** If you can reproduce the bug, move the report to this status.
+   If you created a reproduction project, include the link to the project in
+   your comment.
+
+.. topic:: Example
+
+    Here is a sample comment for a bug report that could be reproduced:
+
+    .. code-block:: text
+
+        Thank you @weaverryan for creating this bug report! This indeed looks
+        like a bug. I reproduced the bug in the "kernel-bug" branch of
+        https://github.com/webmozart/symfony-standard.
+
+        Status: Reviewed
+
+The Pull Request Review Process
+-------------------------------
+
+The process for reviewing pull requests (PRs) is similar to doing a review of a
+bug report. Reviews of pull requests usually take a little longer since you need
+to understand the functionality that has been fixed or implemented and then find
+out whether the implementation is complete.
+
+It is okay to do partial reviews. If you do a partial review, post how far you
+got and leave the PR in "Needs Review" state.
+
+Pick a pull request from the `PRs in need of review`_ and follow these steps:
+
+1. **Is the PR Complete**?
+
+   Every pull request must contain a header that gives some basic information
+   about the PR. You can find the template for that header in the
+   `Contribution Guidelines`_.
```

Comment:
IIRC, we have an anchor for the section that contains the template and and can just link to this directly. Otherwise, we should at least use the `doc` role here for internal links.